mmv1/third_party/terraform/services/apphub/data_source_apphub_discovered_workload_test.go

```diff
@@ -0,0 +1,146 @@
+package apphub_test
+
+import (
+    "testing"
+
+    "github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+    "github.com/hashicorp/terraform-provider-google/google/acctest"
```

Comment:
Can you add "github.com/hashicorp/terraform-provider-google/google/envvar" to fix the unit tests?

Reply:
Done.